### DISCUSSION TITLE
SyntaxError: unexpected token .

### DISCUSSION THREAD

**Discussion by s-titov** [radix-ui/primitives#3284] [Help] [answered]
I'm using the react-dialog and react-accordion primitives with Next.js 14.2, but I've encountered an error in an older version of Chrome on Android (74). How can I fix this issue?

**Comment by georgeeburt** [comment]
Could you share the specific error you're seeing? This would help identify whether it's related to the Dialog, Accordion, or some underlying browser API compatibility issue.​​​​​​​​​​​​​​​​

**Reply by s-titov** [reply]
> Could you share the specific error you're seeing? This would help identify whether it's related to the Dialog, Accordion, or some underlying browser API compatibility issue.​​​​​​​​​​​​​​​​

They both had the same error. 

![image](https://github.com/user-attachments/assets/eb6d1c46-b576-4fbb-bcba-67eb38109f53)

**Comment by s-titov** [comment, marked as answer]
I've fixed it. Just added this to my next config:
```javascript
  transpilePackages: [
    '@radix-ui',
  ],
```
